.github/workflows/ci.yaml

```diff
@@ -0,0 +1,88 @@
+name: Continuous Integration
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+    paths:
+      - .github/workflows/ci.yml
+      - "oculy/**"
+      - pyproject.toml
+
+jobs:
+  formatting:
+    name: Check code formatting
+    runs-on: windows-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.9
+      - name: Install tools
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r lint_requirements.txt
+      - name: Isort
+        run: |
+          isort src\oculy -c;
+      - name: Black
+        if: always()
+        run: |
+          black src\oculy --check;
+      - name: Flake8
+        if: always()
+        run: |
+          flake8 src\oculy;
+  tests:
+    name: Unit tests
+    runs-on: ${{ matrix.os }}
+    needs:
+      - formatting
+    if: needs.formatting.result == 'success'
+    strategy:
+      matrix:
+        os: [windows-latest]
+        python-version: ["3.9", "3.10"]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r test_requirements.txt
+      - name: Install project
+        run: |
+          pip install -e .
+      - name: Test with pytest
+        run: |
+          pip install pytest-cov
+          python -X dev -m pytest --pyargs oculy --cov oculy --cov-report xml -v
+      - name: Upload coverage to Codecov
+        uses: codecov/codecov-action@v1
+        with:
+          token: ${{ secrets.CODECOV_TOKEN }}
+          flags: unittests
+          name: codecov-umbrella
+          fail_ci_if_error: true
+
+  # Added to summarize the matrix (otherwise we would need to list every single
```

Comment:
Using bors is not really worth it here so you can get rid of that part actually.

Reply:
Actually, I haven't used any *bors* in the oculy package and it was a comment remained from the repository **pyvisa**

Reply:
Pyvisa uses bors but it is way overkill given the traffic. Just remove that last part.

Reply:
👍